### ISSUE TITLE
Question/Request regarding LimbControl and the compatbility with GoLoco

### ISSUE THREAD

**Issue by BenjiThatFoxGuy** [Dreadrith/LimbControl#1]
Hey! Can we get something more than just an editor script? It doesn't play nice with Goloco and even without a prefab - if we got instructions how to set it up manually someone could make a VRCFury prefab version which would then mean it would play nicely with systems like GoLoco because VRCFury would be the one responsible for merging everything as needed.

**Comment by manafoo** [comment]
yep , it dosn't work with vrcfury goloco prefab , because of the base and action... etc, in the script,

**Comment by staticanimal** [comment]
seconding this, it'd be nice to have some kind of official way to use both.
i know it's possible since i've been able to get both of them to work simultaneously on an unmodified version of [this avatar](https://payhip.com/b/2YnMa) which for some reason ships with what appears to be a just slightly modified enough version of gogoloco.

edit: i realize the version of gogoloco that ships with this avatar might just be an older one. i don't recall the version number being listed anywhere in the documentation though.

**Comment by staticanimal** [comment]
UPDATE: through a lot of trial and error, i've been able to make both GoGoLoco and LimbControl function simultaneously. instructions below. [this tutorial assumes the user already has prior knowledge of the vchat sdk and its project folder structures]

<img width="1319" height="519" alt="Image" src="https://github.com/user-attachments/assets/d42806fb-096d-4216-975b-c54e69227a9b" />
